src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -189,9 +234,16 @@
 	public static final RegistryObject<Item> CLAW_OF_NRUBYIGLITH = REGISTER.register("claw_of_nrubyiglith", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.HORRORTERROR_TIER, 6, -2.2F).efficiency(4.0F).disableShield().set(MSItemTypes.SICKLE_TOOL).add(OnHitEffect.HORRORTERROR), new Item.Properties().tab(MSItemGroup.WEAPONS).rarity(Rarity.UNCOMMON)));
 	public static final RegistryObject<Item> CANDY_SICKLE = REGISTER.register("candy_sickle", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.CANDY_TIER, 6, -2.2F).efficiency(2.5F).disableShield().set(MSItemTypes.SICKLE_TOOL).add(OnHitEffect.SET_CANDY_DROP_FLAG), new Item.Properties().tab(MSItemGroup.WEAPONS)));
 	
+	
+	//Scythes
 	public static final RegistryObject<Item> SCYTHE = REGISTER.register("scythe", () -> new WeaponItem(new WeaponItem.Builder(Tiers.IRON, 4, -2.6F).efficiency(1.5F).disableShield().set(MSItemTypes.SICKLE_TOOL), new Item.Properties().tab(MSItemGroup.WEAPONS)));
 	public static final RegistryObject<Item> EIGHTBALL_SCYTHE = REGISTER.register("eightball_scythe", () -> new WeaponItem(new WeaponItem.Builder(Tiers.IRON, 3, -2.6F).efficiency(1.5F).disableShield().set(MSItemTypes.SICKLE_TOOL).add(OnHitEffect.RANDOM_DAMAGE).set(ItemRightClickEffect.EIGHTBALL), new Item.Properties().defaultDurability(600).tab(MSItemGroup.WEAPONS)));
-	
+	public static final RegistryObject<Item> TIME_FLAYER = REGISTER.register("time_flayer", () -> new WeaponItem(new WeaponItem.Builder(Tiers.DIAMOND, 3, -2.6F).efficiency(1.5F).disableShield().set(MSItemTypes.SICKLE_TOOL).add(OnHitEffect.TIME_SLOWNESS_AOE).add(OnHitEffect.RANDOM_DAMAGE), new Item.Properties().tab(MSItemGroup.WEAPONS).rarity(Rarity.RARE)));
```

Comment:
Is this intended to be a Time aspect weapon?

Reply:
yes! did i miss s0mething? i kind 0f assumed that time sl0wness a0e includes requiring the time aspect by default, is this n0t the case?

Reply:
It is true that the the time slowness AOE is time aspect specific, but it is already used by the Time aspect Denizen weapon Fear No Anvil. We currently try to make an Aspect weapon accessible to the player before Denizen weapons which has different functionality from the Denizen weapon. For example with the Breath aspect, there is the Zephyr Cane for the Aspect weapon(propels the player forwards) and the Typhonic Trivializer for the Denizen weapon(makes target(s) fly up into the air)

Reply:
If you can come up with a new idea that you think would be fun, we can work together to code it but I dont think we should reuse in this case!